stress/shell/src/main/java/alluxio/stress/cli/RegisterWorkerBench.java

```diff
@@ -106,10 +106,14 @@ public void prepare() throws Exception {
     }
     mBlockCount = blockCount;
 
-    // Prepare these block IDs concurrently
-    LOG.info("Preparing blocks at the master");
-    RpcBenchPreparationUtils.prepareBlocksInMaster(blockMap, getPool(), mParameters.mConcurrency);
-    LOG.info("Created all blocks at the master");
+    // The preparation is done by the invoking shell process to ensure the preparation is only
+    // done once, so skip preparation when running in job worker
+    if (!mBaseParameters.mDistributed) {
+      // Prepare these block IDs concurrently
+      LOG.info("Preparing blocks at the master");
+      RpcBenchPreparationUtils.prepareBlocksInMaster(blockMap);
+      LOG.info("Created all blocks at the master");
+    }
```

Comment:
Why do I see changes from other people's PR here? I only see one commit in this PR but it includes code from other PR. I think you have to manually remove irrelevant code from your commit in this case...

Reply:
Thanks for your review. Maybe because I resolve the conflict using `rebase and amend`. Now I reset code and apply the patch.